### ISSUE TITLE
bad a11y check in MagnetInteractionCueNode

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/faradays-law#154]
MagnetInteractionCueNode.js

`44   var a11yEnabled = phet.chipper.queryParameters.accessibility;`

That's going to be incorrect when `Sim( { accessibility: true } )`. It should be:

`44   var a11yEnabled = phet.chipper.queryParameters.accessibility;`

**Comment by pixelzoom** [comment]
Fixed above, @zepumph please verify.
